setup.py

```
@@ -146,7 +146,7 @@ def cargo_build():
         else:
             del env['CXX']
     subprocess.check_call(
-        cargo_args + ['--features', 'confstr-ffi'],
+        cargo_args + ['--features', 'confstr-ffi,insecure-skip-verify'],
```

Ruff (0.15.13)

[warning] Consider iterable unpacking instead of concatenation

Replace with iterable unpacking

(RUF005)